test/JDBC/expected/BABEL-3395-vu-verify.out

```diff
@@ -0,0 +1,811 @@
+GO
+~~START~~
+nvarchar
+[Anikait Agrawal               ]
+~~END~~
+
+~~START~~
+nvarchar
+[Anikait Agrawal               ]
+~~END~~
+
+~~START~~
+nvarchar
+[Anikait Agrawal               ]
+~~END~~
+
+~~START~~
+nvarchar
+[Anikait Agrawal               ]
+~~END~~
+
+~~START~~
+nvarchar
+[Anikait Agrawal               ]
+~~END~~
+
+~~START~~
+nvarchar
+[Anikait Agrawal               ]
+~~END~~
+
+~~START~~
+nvarchar
+[Anikait Agrawal               ]
+~~END~~
+
+
+DECLARE @class CHAR(30) = N'Anikait Agrawal'
+SELECT '[' + @class + ']' WHERE (@class) = N'Anikait Agrawal'
+SELECT '[' + @class + ']' WHERE upper(@class) = N'Anikait Agrawal'
+SELECT '[' + @class + ']' WHERE upper(@class) = N'ANIKAIT AGRAWAL'
+SELECT '[' + @class + ']' WHERE upper(@class) = N'Anikait Agrawal '
+SELECT '[' + @class + ']' WHERE lower(@class) = N'Anikait Agrawal'
+SELECT '[' + @class + ']' WHERE lower(@class) = N'anikait agrawal'
+SELECT '[' + @class + ']' WHERE lower(@class) = N'Anikait Agrawal '
+GO
+~~START~~
+varchar
+[Anikait Agrawal               ]
+~~END~~
+
+~~START~~
+varchar
+[Anikait Agrawal               ]
+~~END~~
+
+~~START~~
+varchar
+[Anikait Agrawal               ]
+~~END~~
+
+~~START~~
+varchar
+[Anikait Agrawal               ]
+~~END~~
+
+~~START~~
+varchar
+[Anikait Agrawal               ]
+~~END~~
+
+~~START~~
+varchar
+[Anikait Agrawal               ]
+~~END~~
+
+~~START~~
+varchar
+[Anikait Agrawal               ]
+~~END~~
+
+
+DECLARE @class NCHAR = N'Anikait Agrawal'
+SELECT '[' + @class + ']' WHERE upper(@class) = N'ANIKAIT AGRAWAL'
+SELECT '[' + @class + ']' WHERE lower(@class) = N'anikait agrawal'
+GO
+~~START~~
+nvarchar
+~~END~~
+
+~~START~~
+nvarchar
+~~END~~
+
+
+DECLARE @class CHAR = N'Anikait Agrawal'
+SELECT '[' + @class + ']' WHERE upper(@class) = N'ANIKAIT AGRAWAL'
+SELECT '[' + @class + ']' WHERE lower(@class) = N'anikait agrawal'
+GO
+~~START~~
+varchar
+~~END~~
+
+~~START~~
+varchar
+~~END~~
+
+
+-- different result from TSQL, should be fixed under BABEL-4807
+declare @string1 varchar(30) = N'比尔·拉' COLLATE chinese_prc_ci_as
+SELECT UPPER(@string1)
+SELECT UPPER(@string1) COLLATE chinese_prc_ci_as
+SELECT LOWER(@string1)
+SELECT LOWER(@string1) COLLATE chinese_prc_ci_as
+go
+~~START~~
+varchar
+??·?
+~~END~~
+
+~~START~~
+varchar
+比尔·拉
+~~END~~
+
+~~START~~
+varchar
+??·?
+~~END~~
+
+~~START~~
+varchar
+比尔·拉
+~~END~~
+
+
+declare @string1 char(30) = N'比尔·拉' COLLATE chinese_prc_ci_as
+SELECT '|' + UPPER(@string1) + '|'
+SELECT '|' + UPPER(@string1) + '|' COLLATE chinese_prc_ci_as
+SELECT '|' + LOWER(@string1) + '|'
+SELECT '|' + LOWER(@string1) + '|' COLLATE chinese_prc_ci_as
+go
+~~START~~
+varchar
+|??·?                          |
+~~END~~
+
+~~START~~
+varchar
+|??·?                          |
+~~END~~
+
+~~START~~
+varchar
+|??·?                          |
+~~END~~
+
+~~START~~
+varchar
+|??·?                          |
+~~END~~
+
+
+SELECT UPPER(NULL)
+SELECT LOWER(NULL)
+GO
+~~START~~
+text
```

Comment:
`UPPER` and `LOWER` function should always return `varchar` or `nvarchar`.

Reply:
Done, added wrapper